src/rust/codegen.rs

```diff
@@ -723,6 +729,14 @@ pub fn gen_get_phys_eip(ctx: &mut JitContext, address_local: &WasmLocal) {
 
     ctx.builder.get_local(&address_local);
     ctx.builder.call_fn1_ret("get_phys_eip_slow_jit");
+
+    ctx.builder.get_local(&address_local);
+    ctx.builder.xor_i32();
+    ctx.builder.const_i32(unsafe { memory::mem8 } as i32);
+    ctx.builder.add_i32();
+    ctx.builder.get_local(&address_local);
+    ctx.builder.xor_i32();
+    
```

Comment:
I think this could be moved into `get_phys_eip_slow_jit` (keeps the generated code size small).

Reply:
moved